src/stytch-provider.ts

```diff
@@ -72,10 +190,18 @@ function StytchProvider(this: any, options: StytchProviderOptions) {
     let project_id = res.keymap.project_id.value
     let secret = res.keymap.secret.value
 
+    // avoid the cost of establishing a new connection with
```

Comment:
To indicate that this is a real comment, not just tmp for debugging, format as a full sentence:

```
// Avoid ...

```